index.html

```diff
@@ -18,52 +18,49 @@
 		<div class="grid-container">
 			<div>
 				
-		<div class="header">Home Page</div>
+	    <div class="header"><img src="images/LOGO2.jpg" alt="iMages"></div>
 			
 		<div>
 			
 	<!--This class is for my navigation/menu-->
     <div class="nav">
 		<ul>
-			<li><a href="index_v1.html">Home | </a></li>
-			<li><a href="CommunityService.v1.html">Community service | </a></li>
+			<li><a href="index.v2.html">Home | </a></li>
+			<li><a href="CommunityService.html">Community service | </a></li>
 			<li><a href>Outdoor activities |</a>
 			    <ul>
-					<li><a href="indexpage2.v1.html">Adrenaline Forest and Te Rauparaha Arena</a>
-					<li><a href="indexpage3.v1.html">Rangituhi Walk</a>
-					<li><a href="indexpage4.v1.html">Wainuiomata Camp pt1</a>
-					<li><a href="indexpage5.v1.html">Wainuiomata Camp pt2</a>
+					<li><a href="indexpage2.v2.html">Adrenaline Forest and Te Rauparaha Arena</a>
+					<li><a href="indexpage3.html">Rangituhi Walk</a>
+					<li><a href="indexpage4.html">Wainuiomata Camp pt1</a>
+					<li><a href="indexpage5.html">Wainuiomata Camp pt2</a>
 				  </ul>
 				  </li>
-			     <li><a href="Contact.v1.html">Contact |</a>
+			     <li><a href="Contact.html">Contact |</a>
 			    </ul> 
-            </div>		
-	
+            </div>	
 
 			<div>
 			<div class="box2"></div>
 			
 			<div class="box3"></div>
 			</div>
-			
-			<div class="sidebar">sidebar</div>
-		
 		
-			<div class="content">	
-			   <div class="scroll-area">
-				   
-		<!-- Setup Class to hold and control Radio input Buttons to move each image. -->
+		   
+		<div class="content">	
+			<div class="scroll-area">
+			
+				<!-- Setup Class to hold and control Radio input Buttons to move each image. -->
         <div class="slider">
 		   <div class="images">
 		      <input type="radio" name="slide" id="img1" checked> <!-- Radio is the style of the input buttons. Checked activates the CSS -->
               <input type="radio" name="slide" id="img2"> <!-- ID Controls each separate image through the CSS using a # -->
 			  <input type="radio" name="slide" id="img3">
 			  <input type="radio" name="slide" id="img4">
 			  
-			  <img src="Images/ImageCarousel1.jpg" class="m1" alt="iMAGE"> <!-- Class controls  each separate image through CSS -->
-			  <img src="Images/ImageCarousel1.jpg" class="m2" alt="iMAGE">
-			  <img src="Images/ImageCarousel1.jpg" class="m3" alt="iMAGE">
-			  <img src="Images/ImageCarousel1.jpg" class="m4" alt="iMAGE">
+			  <img src="images/iMAGE.jpg" class="m1" alt="iMAGE"> <!-- Class controls  each separate image through CSS -->
+			  <img src="images/iMAGE.jpg" class="m2" alt="iMAGE">
+			  <img src="images/iMAGE.jpg" class="m3" alt="iMAGE">
+			  <img src="images/iMAGE.jpg" class="m4" alt="iMAGE">

```

Comment:
On the image carousel I got rid of the image carousle box layout as it wasn't fitting the space and the writing was too small so I deleted that image and made another one so when students/end users are lookng at the image carousel it would show them a little about what it's there for.